wagtailio/core/blocks.py

```diff
@@ -188,20 +188,25 @@ class Meta:
         template = "core/blocks/home_page_block.html"
 
 
-class CTABlock(blocks.StructBlock):
-    cta_text = blocks.CharBlock(label="CTA text", max_length=255)
-    cta_page = blocks.PageChooserBlock(label="CTA page", required=False)
-    cta_url = blocks.URLBlock(label="CTA URL", required=False)
+class CTALinkMixin(blocks.StructBlock):
+    def is_required(self):
```

Comment:
Praise: Great work here Victor, this is very clean and clear overall, I really appreciate the effort you've put in here!

Small suggestion: I would personally name this to `is_link_required`, is_required doesn't convey to another developer what is actually being required without digging through the code to the superclass. While this is not an issue now (we can just scroll up a few lines!), if we think into the future when this project as a whole gets more complicated it's small details like this that make a project easier to work on.

Reply:
Makes sense! updated in 128de66